admin/css/admin.css

```diff
@@ -427,6 +427,14 @@ body.post-type-volunteer_opp .wp-pointer-arrow {
 }
 
 /**
+* Widget styling
+*/
+.wi-widget-block-label:after {
```

Comment:
Thanks for explaining and I think your approach will work totally fine as long as it's cross-browser compliant.